### ISSUE TITLE
Implement MSE Loss with tensorflow and pytorch

### ISSUE THREAD

**Issue by pavanreddyml** [pavanreddyml/adversarial-lab#9]
Implement a Mean Squared Error Loss with tf and torch similar to 'adversarial_lab/core/losses/categorical_cross_entropy.py'. The losses must be compatible with autograd and autodiff
